### DISCUSSION TITLE
成立人工智慧實驗室，建置AI影像庫

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#722] [General]
https://futurecity.cw.com.tw/article/2800
人的大老是有其極限得˙，這時將所有數據處存就顯得十分重要了

**Comment by Valentinaa0214** [comment]
AI在醫學方面的幫助實在是進步的越來越多了，將來醫院必定能夠全面人工智能化。
它不僅僅是提升了醫生在診斷時的效率與準確度，更幫助了病人們節省治療的時間與降低風險。

**Comment by HsuChen88** [comment]
使用AI簡化醫院中資料處理的公司有很多，像是Nuance Community的有提供DAX (Dragon Medical Advisors Exchange) 解決方案，能輔助醫療人員進行健康紀錄和做決策。
